Tools/clinic/mypy.ini

```diff
@@ -1,12 +1,15 @@
 [mypy]
+files = Tools/clinic/
+pretty = True
+
 # make sure clinic can still be run on Python 3.10
 python_version = 3.10
-pretty = True
-enable_error_code = ignore-without-code
-disallow_any_generics = True
+
+# be strict...
+strict = True
 strict_concatenate = True
-warn_redundant_casts = True
-warn_unused_ignores = True
-warn_unused_configs = True
+enable_error_code = ignore-without-code,redundant-expr
 warn_unreachable = True
-files = Tools/clinic/
+
+# ...except for one extra rule we can't enable just yet
+warn_return_any = False
```

Comment:
Keep the issue open until this rule has been removed?

Reply:
Sure!